### PR TITLE
Fix OpenShift plugin test after latest changes

### PR DESCRIPTION
**Pull request by SkorikSergey**
### What does this PR do?
After changes on the Theia side we need to fix some locators and rewrite some calls of commands for the OpenShift plugin connector. Also we have some changes in the behavior of OpenshiftPlugin. We need to adapt dependent PageObjects and change some steps in the tests

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/18543

### CODE REVIEW

**Comment by SkorikSergey** [comment]
[ci-test]

**Comment by SkorikSergey** [comment]
[ci-test]

**Comment by SkorikSergey** [comment]
[crw-ci-test]

File: tests/.infra/centos-ci/nightly/cico-openshift-connector-test.sh [hunk 1]
```diff
@@ -17,7 +17,7 @@ function runOpenshiftConnectorTest(){
     mkdir report
     REPORT_FOLDER=$(pwd)/report
     ### Run tests
-    docker run --net=host  --ipc=host -v $REPORT_FOLDER:/tmp/e2e/report:Z -p 5920:5920 \
+    docker run --net=host  --ipc=host -v $REPORT_FOLDER:/tmp/e2e/report:Z --shm-size=1g \
```
Comment (Ohrimenko1988) [inline tests/.infra/centos-ci/nightly/cico-openshift-connector-test.sh:20]: Do ports expose unnecessarily?

**Review by Ohrimenko1988 — APPROVED** [review]

**Pull request merged**